deploy/helm/templates/rbac/cluster_pod_required_role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeblocks-cluster-pod-role
+  name: {{ include "kubeblocks.fullname" . }}-cluster-pod-role
```

Comment:
the name of this cluster role must be "kubeblocks-cluster-pod-role" because the addon repo hard-coded its name...

https://github.com/apecloud/kubeblocks-addons/blob/main/addons-cluster/kblib/templates/_rbac.tpl#L52

![image](https://github.com/user-attachments/assets/52fc0dbc-0b0e-4d0f-9fcc-9718d02149aa)